compiler/runtime/Runtime.hpp

```diff
@@ -327,7 +327,11 @@ typedef enum
    TR_MethodCallAddress                   = 99,
    TR_DiscontiguousSymbolFromManager      = 100,
    TR_ResolvedTrampolines                 = 101,
-   TR_NumExternalRelocationKinds          = 102,
+   TR_InlinedStaticMethod                 = 102,
+   TR_InlinedSpecialMethod                = 103,
+   TR_InlinedAbstractMethod               = 104,
```

Comment:
Is there a reason why inlined interface methods do not need their own relocation kind ?  I thought I would ask since they have their own call bytecode (analogous to static/special/abstract) as well as their own api that is based on the CHTable (for single interface implementor).

Reply:
They already existed prior to this change: https://github.com/eclipse/omr/blob/9c792c13b65a2fbabc0e9638730e8a93344dbb6f/compiler/runtime/Runtime.hpp#L282-L283

Why the others weren't added back when these were is not something I can answer as this was done before my time :)

Reply:
For context, there's two types of inlined method relocations:

1. Inlined method with NOP guard
2. Inlined method

First is used when the inlined call site has a guard, the second is used when there is no guard. I've added the new types so that each of the "with NOP guard" relocations have an associated "without NOP guard" relocation.

Reply:
Thanks for the explanation.